server/lib/notifications.js

```diff
@@ -315,5 +315,14 @@ async function notifyByEmail(activity) {
         template: 'collective.created.opensource',
       });
       break;
+
+    case activityType.BACKYOURSTACK_DISPATCH_CONFIRMED:
+      activity.data.orders.map(async order => {
```

Comment:
Does that work? I'm not aware of Array.map being async.

I understand this can technically work, but the syntax should make clear the behavior is understood and not confuse future developers.